vendor/github.com/v3io/v3io-go/pkg/dataplane/streamconsumergroup/config.go

```diff
@@ -52,3 +55,13 @@ func NewConfig() *Config {
 
 	return c
 }
+
+// returns a logger function according to the set log level
+// we want this to ensure the logger used, overrides the severity level set by the parent logger
+// so, by default, LogLevel is set to 0, and hence persist the debug
+func (c *Config) getLeveledLogger(logger logger.Logger) func(interface{}, ...interface{}) {
```

Comment:
You are essentially creating only two levels of logs (above 5 and below). This log level feature should allow us to create a wide range of verbosity and therefore each time you call getLeveledLogger you should provide the min level you want to log on.